Amplify/Categories/DataStore/DataStoreCategory+HubPayloadEventName.swift

```diff
@@ -23,4 +23,44 @@ public extension HubPayload.EventName.DataStore {
     /// Dispatched when DataStore receives a sync response from the remote API via the API category. The Hub Payload
     /// will be a `MutationEvent` instance that caused the conditional save failed.
     static let conditionalSaveFailed = "DataStore.conditionalSaveFailed"
+
+    /// Dispatched when network status has changed (active or not)
+    /// The HubPayload will be a boolean value `isActive` indicating the status of network
+    // TODO: networkStatusChanged to be implemented
+    static let networkStatusChanged = "DataStore.networkStatusChanged"
+
+    /// Dispatched on DataStore start and also every time a local mutation is enqueued and processed in the outbox
+    /// HubPayload `OutboxStatusEvent` contains a boolean value `isEmpty` to notify if there are mutations in the outbox
+    static let outboxStatus = "DataStore.outboxStatus"
+
+    /// Dispatched when all the graphql subscriptions estabilished
```

Comment:
- `Dispatched when all of the subscriptions to syncable models have been established`